packages/blade/src/components/Typography/BaseText/BaseText.tsx

```diff
@@ -35,7 +37,9 @@ export type BaseTextProps = {
   lineHeight: keyof Theme['typography']['lineHeights'];
   as?: 'code' | 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'p' | 'span';
   textAlign?: 'center' | 'justify' | 'left' | 'right';
-  children?: React.ReactNode | undefined;
+  truncateAfterLines?: number;
+  className?: string;
```

Comment:
Shouldn't this have `_` in prefix?
`_className`

Reply:
Nupe since no one is explicitly adding this class(yes consumers can abuse this), it's styled-component that is adding this class. also this is anyways `BaseText` whose job is to let people customise things when our regular text doesn't fits their need. None of the other Text components have this prop so it's safe.